tests/test_contrib_aiohttp.py

```diff
@@ -150,3 +151,17 @@ class TestAIOHTTP:
             await wait_for(collected)
 
         assert collected == []
+
+    async def test_client_closed_correctly(self):
+        # Actual behaviour of the web.run_app clean up phase is
+        # to cancel all active tasks at the finally block
+        # https://git.io/fj56P
+
+        tasks = asyncio.all_tasks()
```

Comment:
`.all_tasks()` introduced in Python 3.7 https://docs.python.org/3/library/asyncio-task.html?highlight=all_tasks#asyncio.all_tasks. Package supports Python 3.6 too, fix test please.

Reply:
Yes sure, thank you